.dockerignore

```diff
@@ -0,0 +1,5 @@
+.git
```

Comment:
This .dockerignore file is optional, I just stumbled across it in a Docker "best practices" document. I guess it saves some space in the container if we explicitly mark these files as excluded?

Reply:
Nice!